pkg/charts/linkerd2/values_test.go

```diff
@@ -4,7 +4,9 @@ import (
 	"reflect"
 	"testing"
 
+	appv1 "k8s.io/api/apps/v1"
```

Comment:
You'll need to remove these unused imports.

Reply:
Done.